classes/notification/managerDelegate/PendingRevisionsNotificationManager.inc.php

```diff
@@ -43,8 +43,8 @@ public function getNotificationUrl($request, $notification) {
 		$stageData = $this->_getStageDataByType();
 		$operation = $stageData['path'];
 
-		import('lib.pkp.controllers.grid.submissions.SubmissionsListGridCellProvider');
-		return SubmissionsListGridCellProvider::getUrlByUserRoles($request, $submission, $notification->getUserId(), $stageData['path']);
+		import('classes.core.ServicesContainer');
+		return ServicesContainer::instance()->getWorklowUrlByUserRoles($submission, $notification->getUserId(), $stageData['path']);
```

Comment:
Good catch again! THere were a bunch of issues there, fixed in: https://github.com/NateWr/pkp-lib/commit/18e4ad4d0e80b9329f6007de8eda76fa89201091